pytx/scripts/get_compromised_credentials.py

```diff
@@ -80,15 +75,16 @@ def get_parser():
     parser.add_argument('-o', '--output')
     parser.add_argument('-s', '--since')
     parser.add_argument('-u', '--until')
+    parser.add_argument('-l', '--limit')
 
     return parser
 
 if __name__ == '__main__':
     args = get_parser().parse_args()
-    if args.output is not None:
-        handle = open(args.output, 'w')
-    else:
-        handle = None
+    
+    handle = open(args.output, 'w') if args.output else None
```

Comment:
Although this makes the code fit on fewer lines, it makes it, in my opinion, more difficult to read.